channel-sdk/src/main/java/com/zello/channel/sdk/Session.kt

```diff
@@ -169,7 +170,9 @@ class Session internal constructor(
 	 * @param message the message to send
 	 */
 	fun sendText(message: String) {
-		// TODO: Implement sendText(message:)
+		if (!initialized) return
```

Comment:
Leaving it to your judgement, but it might be useful to have some kind of warning in this case (or even throw an exception). Of course, if some dev finds themselves at the point of trying to call `sendText` after the native libraries failed to load, a lot of things must have gone awry to get to that point, so maybe we would consider it overkill

Reply:
`connect()` fails if the native library load failed, so these are just sanity checks. `connect()` should provide more information than just a boolean failure value though.

Reply:
Gotcha; that makes sense